contracts/assembly/src/contract.rs

```diff
@@ -653,26 +653,9 @@ pub fn query(deps: Deps, _env: Env, msg: QueryMsg) -> StdResult<Binary> {
 }
 
 /// Returns proposal information by id.
-pub fn query_proposal(deps: Deps, proposal_id: u64) -> StdResult<ProposalResponse> {
+pub fn query_proposal(deps: Deps, proposal_id: u64) -> StdResult<Proposal> {
```

Comment:
Can we remove this function and place `PROPOSALS.load(deps.storage, proposal_id)?` directly in query endpoint?

Reply:
Yes, done